html/img_import/credentials.inc.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+# DB credentials
+define("DBHOST", "localhost");
+define("DBNAME", "humanface");
+define("DBUSER", "humanface");
+define("DBPASS", "humanface");
+
+?>
```

Comment:
"credential.inc.php" shouldn't be in the Github repo. Include "credential.inc.php" in "html/" not but copying it into the "img_import" folder.